src/components/Layout/Header/Header.tsx

```diff
@@ -194,27 +241,35 @@ const HeaderContent: FunctionComponent<
                             </div>
                             {!props.minimal && (
                                 <>
-                                    <div className="hidden flex-1 md:ml-4 lg:block">
+                                    <div className="hidden flex-1 items-center self-center md:ml-4 lg:block">
```

Comment:
Seeing these drop downs built out and they're looking a little too airy. I've adjusted padding on each dropdown in the Figma file: https://www.figma.com/design/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=3279-21067&focus-id=14131-40010&m=dev

Reply:
@gitstart-sourcegraph One more tweak to the padding on the Integrations section! Then we can stop fiddling with it! lol. See adjustment here: https://www.figma.com/design/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=3279-21067&version-id=6195991730&focus-id=14131-40010&m=dev